Part2_search_selection.R

```diff
@@ -322,14 +326,8 @@ litsearchr::write_search(grouped,
                          verbose = TRUE,
                          closure = "left")
 
-# Section 4: build PRISMA figure (Xiaoxia)
-library(PRISMA2020)
-
-
-## ----prisma--------------------------------------------------------------
-#install.packages("PRISMAstatement")
-library(PRISMAstatement)
-
+# Section 4: build PRISMA figure (Xiaoxia) ----
```

Comment:
I think this section is unnecessarily long. It would be enough to show a full example with all/most customisation and refer the reader to the package documentation.